core/src/test/java/org/axonframework/eventsourcing/eventstore/EmbeddedEventStoreTest.java

```diff
@@ -69,7 +69,7 @@ public void setUp() {
     private void newTestSubject(int cachedEvents, long fetchDelay, long cleanupDelay) {
         Optional.ofNullable(testSubject).ifPresent(EmbeddedEventStore::shutDown);
         testSubject = new EmbeddedEventStore(storageEngine, NoOpMessageMonitor.INSTANCE, cachedEvents, fetchDelay,
-                                             cleanupDelay, MILLISECONDS);
+                                             cleanupDelay, MILLISECONDS, SnapshotResolver.lastSnapshotResolver());
```

Comment:
This stuff would benefit..from a Builder maybe, with defaults in it. What do you think? ;p